### DISCUSSION TITLE
Development team chat

### DISCUSSION THREAD

**Discussion by caincoulton** [caincoulton/virtualdomains#2] [General]
Hey @tkempf @britzke, let's continue and discussion around Virtual Domains and it's development here

**Comment by tkempf** [comment]
Ok, i checked out your latest code, built it on Windows and tried to install it on a linux server running apache. Trying to install the package throws Errors (see attached Screenshot). 
This is probably because of the case-sensitive filenames on a linux server.

![Screenshot (46)](https://github.com/caincoulton/virtualdomains/assets/2909428/90e20362-b7b2-49b6-9d44-04f5766391d8)
